projects/angular-cld/src/lib/cloudinary.service.ts

```diff
@@ -15,6 +15,10 @@ const isJsonLikeString = function (str: any): boolean {
   return str && typeof str === 'string' && (str.trim().match(/^{[\s\S]*?}$/) !== null);
 };
 
+const isArrayLikeString = function (str: any): boolean {
```

Comment:
Why are we using Any here? I see it in various other places as well,
we really don't know what's coming in?

Reply:
Its a string but I wanted to keep the same structure as the other methods. I'll make a separate pr for the rest of the types and verify that its not breaking anything